contributors/devel/welcome-to-kubernetes-new-developer-guide.md

```diff
@@ -0,0 +1,126 @@
+bewildered by the thousands of [open
+issues](https://github.com/kubernetes/kubernetes/issues) in our main repository.
+But now that you know about SIGs, it's easy to filter by labels to see what's
+going on in your SIG. For more information about our issue system, check out
+[issues.md](https://github.com/kubernetes/community/blob/master/contributors/devel/issues.md).
+
+Downloading, Building, and Testing Kubernetes
+---------------------------------------------
+
+This guide is non-technical, so I'm going to refer you to our online
+documentation.We have plenty of documentation available under
+[github.com/kubernetes/community/contributors/devel](https://github.com/kubernetes/community/tree/master/contributors/devel).
+Check out
+[development.md](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md)
+for more details.
+
+Pull-Request Process
+--------------------
+
+When you submit a pull request to
+[kubernetes/kubernetes](https://github.com/kubernetes/kubernetes), the Three
+Kubernetes Bots will show up and start applying labels and assigning reviewers.
+Our bots live in
+[github.com/kubernetes/test-infra](https://github.com/kubernetes/test-infra) and
+their names are:
+
+ 1. @k8s-bot
+ 2. @k8s-ci-robot
+ 3. @k8s-merge-robot
+
+The Continuous Integration (CI) robot, @k8s-ci-robot, won't run the tests until
+a Kubernetes maintainer gives the OK. We require PRs to be approved for testing
+in order to prevent people from running malicious code on our CI servers, for
+example mining for bitcoins.
+
+Usually someone will approve testing and review your PR within a day or two.
+Have some patience, of course; Kubernetes is a huge, semi-chaotic project and
+people are often busy debugging issues or preparing for the next release. If
+your PR languishes unattended for more than a couple days, @mention someone in
+the
+[OWNERS](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md)
+file of the your changes' root directory and you should get a quicker response.
+
+In order for your PR to be merged, you need four things. When you have all four,
+the @k8s-merge-robot will automatically merge your commits.
+
+ 1. CNCF-CLA verification. You must sign the Contributor License Agreement to
+    contribute to Kubernetes. Check out our FAQ at
+    [https://github.com/kubernetes/kubernetes/wiki/CLA-FAQ](https://github.com/kubernetes/kubernetes/wiki/CLA-FAQ).
+    Once you sign the CLA, @k8s-ci-robot should automatically add the "cncf-cla:
+    yes" label.
+ 2. The "approved" label. Get this by having an owner from the OWNERS file
+    comment "/approve" on your PR.
+ 3. The LGTM label. Get this by having a Kubernetes org member write "/lgtm" on
+    your PR.
+ 4. All green CI. If CI fails, the robots will comment with instructions on how
+    to rerun the tests.
+
+There is an entire SIG
+([sig-contributor-experience](https://github.com/kubernetes/kubernetes/wiki/CLA-FAQ))
```

Comment:
Link to the contribex sig doc in k/community instead of the wiki which is out of date.